### DISCUSSION TITLE
Lumen on The Changelog?!

### DISCUSSION THREAD

**Discussion by jerodsanto** [GetFirefly/firefly#391] [Ideas]
Howdy from [The Changelog](https://changelog.com/podcast)!

_(We've been interviewing the hackers, leaders, and innovators of the software world for over a decade.)_

We'd love to have you on the show sometime soon. Would @KronicDeth and @bitwalker be the best people to talk with us about Lumen and all the cool stuff y'all are up to?

Whattya say? Please email jerod@changelog.com to coordinate. 

Looking forward to it! 💚

**Comment by bitwalker** [comment]
@jerodsanto We’d be happy to! We’ll get in touch :)

**Comment by jerodsanto** [comment]
Y'all are fast! 😆 

_(side note: I'm a big fan of Timex and we use it on changelog.com. I _think_ I've told you this already at some point, but wanted to mention it again because THANKS!)_

**Comment by bcardarella** [comment]
Per Dec 9 standup, this issue has been converted to a Discussion.

**Comment by bcardarella** [comment]
Per Dec 9 standup, this issue has been converted to a Discussion.
